src/volttron/client/commands/install_agents.py

```diff
@@ -103,13 +106,6 @@ def install_agent_directory(opts, publickey=None, secretkey=None):
     #
```

Comment:
I want to leave this comment in there as this feature is not yet ported over to modular. How we deal with this would depend on whether we are going to be using pip/poetry or something else to install packages and manage dependencies